org.nicolasbock.ebuildtester.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: org.nicolasbock.ebuildtester
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: ebuildtester
+finish-args:
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - python3-modules.json
+  - name: ebuildtester
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --verbose .
+      - python setup.py install --prefix /app --verbose
+      - mkdir --parents /app/usr/bin
```

Comment:
having `/app/usr/` is illogical. `/app` is the prefix, like `/usr` would be

Reply:
Yes, admittedly this naming scheme does not conform to the standard filesystem layout. I will address this in the next iteration.